### PR TITLE
Move the `serverless-bench` folder

### PR DESCRIPTION
**Pull request by pPrecel**
**Description**

Changes proposed in this pull request:

- as in the title

**Related issue(s)**

- https://github.com/kyma-project/third-party-images/pull/176

### CODE REVIEW

File: tests/serverless-bench/run-benchmarks.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# running benshmarks
+for FUNCTION in ${ALL_FUNCTIONS[@]}; do 
+    echo "--------------------------------------------------------------------------------"
+    echo "Benchmarking function ${FUNCTION} at URL: http://${FUNCTION}.${TEST_NAMESPACE}.svc.cluster.local"
+    echo "--------------------------------------------------------------------------------"
+
+    locust --headless --users 50 \
+    -r 50 -t 1m \
+    -H "http://${FUNCTION}.${TEST_NAMESPACE}.svc.cluster.local" \
+    -f /home/locust/locust.py --csv "${FUNCTION}" --logfile /dev/null
```
Comment (dbadura) [inline tests/serverless-bench/run-benchmarks.sh:23]: Can we use relative path?

File: tests/serverless-bench/locust.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import logging
+from locust import HttpUser, task, constant, events
+
+expected_string = "Hello Serverless"
+
+class Python38Test(HttpUser):
```
Comment (dbadura) [inline tests/serverless-bench/locust.py:6]: It can be named `StressTest`

File: CODEOWNERS [hunk 3]
```diff
@@ -195,6 +195,9 @@ components/etcd-tls-setup-job/ @PK85 @piotrmiskiewicz @ksputo @szwedm @voigt @wo
 # Function Controller tests
 /tests/function-controller/ @m00g3n @pPrecel @dbadura
 
+# The serverless-bench Tests
+/tests/serverless-bench/ @m00g3n @pPrecel @dbadura @moelsayed
```
Comment (dbadura) [inline CODEOWNERS:199]: `cortney`?

**Review by dbadura — CHANGES_REQUESTED** [review]

File: tests/serverless-bench/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,11 @@
+FROM python:alpine3.15
+WORKDIR /home/locust
+RUN apk update && apk add bash curl jq miller git build-base linux-headers c-ares coreutils \
+&& git clone https://github.com/locustio/locust.git --depth 1 --branch 2.6.0 locust \
+&& cd locust \
+&& pip install . \
```
[the remark this answers is not in the document]
Comment (dbadura) [reply tests/serverless-bench/Dockerfile:6]: I will fix it in separate PR.

**Review by dbadura — APPROVED** [review]

**Review by grego952 — DISMISSED** [review]

**Review by dbadura — APPROVED** [review]

**Review by grego952 — APPROVED** [review]

**Pull request merged**